index.html

```diff
@@ -6,128 +6,133 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./styles.css" />
     <title>Lenz Lösches Homepage</title>
-    <script src="script.js"></script>
+    <script src="script.js" defer></script>
   </head>
   <body id="body">
     <header></header>
     <nav>
-      <div class="navdiv" id="navdiv">
-        <h3 class="navelement"><a href="#body">Home</a> -</h3>
+      <ul class="nav" id="navdiv">
+        <li class="nav__element"><a href="#body">Home</a> -</li>
 
-        <h3 class="navelement"><a href="#lebenslauf">About</a> -</h3>
+        <li class="nav__element"><a href="#lebenslauf">About</a> -</li>
 
-        <h3 class="navelement"><a href="#skills">Skills</a> -</h3>
+        <li class="nav__element"><a href="#skills">Skills</a> -</li>
 
-        <h3 class="navelement"><a href="#links">Links</a> -</h3>
+        <li class="nav__element"><a href="#links">Links</a> -</li>
 
-        <h3 class="navelement"><a href="#kontakt">Contact</a></h3>
+        <li class="nav__element"><a href="#kontakt">Contact</a></li>
 
-        <div class="arrow" id="arrow">
-          <img class="arrowimg" src="bilder/arrow.gif" alt="" />
+        <div class="nav__arrow" id="arrow">

```

Comment:
You should have only `<li>` tags inside your list. Please wrap the `<div>` inside an `<li>`